src/app/newstudents/page.tsx

```diff
@@ -0,0 +1,55 @@
+import { SectionTitle } from "@components/SectionTitle";
+import { InfoCard } from "@components/InfoCard";
+import { STORIES, TODAY_STATISTICS } from "@data";
+import { Sponsors } from "@components/Sponsors";
+import { TitleSubtitle } from "@components/TitleSubtitle";
+
+const NewStudents = () => {
```

Comment:
Otherwise, thanks again for taking a stab at this, and LGTM. We should get this linked on Yalies "you do not have access page". CourseTable FAQs are solid, reflected by the fact that we barely get any messages from them

Reply:
For sure - I can stand up a separate PR for Yalies.io once we get this into main